numbagg/decorators.py

```diff
@@ -995,31 +997,78 @@ def gufunc(self, *, target):
         return vectorize(self.func)
 
 
+# Layer categories and which backends they allow
+_LAYER_CATEGORIES = {
+    "default": {"tbb", "omp", "workqueue"},
+    "safe": {"tbb"},
+    "threadsafe": {"tbb", "omp"},
+    # OpenMP is not fork-safe on Linux, but is on other platforms
+    "forksafe": {"tbb", "workqueue", "omp"}
+    if sys.platform != "linux"
+    else {"tbb", "workqueue"},
+}
+
+# Backends that are safe to use in multi-threaded contexts (e.g., ThreadPoolExecutor)
+_THREADSAFE_BACKENDS = {"tbb", "omp"}
```

Comment:
this is already available as `_LAYER_CATEGORIES["threadsafe"]`, no need to duplicate this

```suggestion
```